student.rb

```diff
@@ -6,7 +6,14 @@ def initialize
     @classroom = classroom
```

Comment:
- [ ] Please add classroom as a parameter here it gives this error 
![image](https://user-images.githubusercontent.com/74744787/217647938-36bff4a1-bc74-44f3-982d-8dd0b0ec6e10.png)